content/200-orm/200-prisma-client/100-queries/058-transactions.mdx

```diff
@@ -10,15 +10,15 @@ tocDepth: 3
 A database transaction refers to a sequence of read/write operations that are _guaranteed_ to either succeed or fail as a whole. This section describes the ways in which the Prisma Client API supports transactions.
 
 - For more in-depth examples and use cases, refer to the 📖 [transactions guide](/orm/prisma-client/queries/transactions).
-- For information about transactions in general and the reasoning behind Prisma's current solutions, see [Blog: How Prisma supports transactions](https://www.prisma.io/blog/how-prisma-supports-transactions-x45s1d5l0ww1).
+- For information about transactions in general and the reasoning behind Prisma ORM's current solutions, see [Blog: How Prisma ORM supports transactions](https://www.prisma.io/blog/how-prisma-supports-transactions-x45s1d5l0ww1).
```

Comment:
I think this is fully outdated and should probably just be removed @nikolasburk?

Reply:
Niko isn't back til Monday, should we wait or delete?

Reply:
Agree, this can be dropped 👍  In that case, because it's a bullet list and there's only one item left in the list, I would also say that the hyphen from the previous sentence can be removed so it's not rendered as a one-item-list.